packages/drivers/odsp-driver/src/vroom.ts

```diff
@@ -89,6 +89,7 @@ export async function fetchJoinSession(
             );
 
             const socketUrl = response.content.deltaStreamSocketUrl;
+            // e.g. url format should be https://{hostName}/fluid
```

Comment:
```suggestion
            // expecting socketUrl to be something like https://{hostName}/fluid
```

Suggesting using language that's more non-committal :)

And maybe also just do https://{hostName}/... unless it really is just /fluid at the end typically